### ISSUE TITLE
Wrong order of last 2 snapshots in bench.cmp

### ISSUE THREAD

**Issue by herenowcoder** [alco/benchfella#44]
Привіт! :)

I recently discovered the parameterless `bench.cmp`, comparing last two snapshots.

This is very cool and intuitive, but! I believe the order of these two snapshots is wrong:

```
% mix bench; mix bench; mix bench.cmp -d percent
Settings:
  duration:      1.0 s

## Extt.Bench
[20:23:25] 1/1: auth

Finished in 2.46 seconds

## Extt.Bench
auth       10000   214.48 µs/op

Settings:
  duration:      1.0 s

## Extt.Bench
[20:23:28] 1/1: auth

Finished in 1.99 seconds

## Extt.Bench
auth       10000   173.41 µs/op

bench/snapshots/2016-02-04_20-23-30.snapshot vs
bench/snapshots/2016-02-04_20-23-27.snapshot

## Extt.Bench
auth    +23.69%
```

You see: my 1st result was worse, 2nd was better, but the report (red-bad in this case) is as it'd be other way around :)

Comparison should go in a way: earlier vs later ...

cheers!

Wojtek

**Comment by alco** [comment]
Great find! I have fixed it in https://github.com/alco/benchfella/commit/6876ba77f540947b9d87b8d217d6109e9e24b558

**Comment by herenowcoder** [comment]
Cool. It works :)

Can you make v0.3.2 with it? It's much more clean than using a commit ref...

cheers,
W.

**Comment by herenowcoder** [comment]
Hi again,
pls hex.publish - it would help a lot :)

**Comment by alco** [comment]
@herenowcoder Published 0.3.2.

**Comment by herenowcoder** [comment]
Thx! [:
